### PR TITLE
Display Auto delete logs next cron schedule in local time

### PR DESCRIPTION
**Pull request by mariadanieldeepak**
Closes https://github.com/sudar/email-log-auto-delete-logs/issues/5, Fixes https://github.com/sudar/email-log-auto-delete-logs/issues/4

### CODE REVIEW

**Comment by mariadanieldeepak** [comment]
@sudar,

> What is the status on this?

I missed this PR because I couldn't find it on Waffle. I added the time to show up on Settings page and Dashboard as well.

File: include/Util/helper.php [hunk 1]
```diff
@@ -131,3 +131,23 @@ function join_array_elements_with_delimiter( $value, $delimiter = ',' ) {
...
+ *
+ * @since 2.3.0
+ */
+function render_auto_delete_logs_next_run_schedule() {
+	?>
+	<?php if ( wp_next_scheduled( 'el_scheduled_delete_logs' ) ) : ?>
+		<p>
+			<?php _e( 'Auto delete logs cron will be triggered next at', 'email-log' ); ?>:
+			<?php $date_time_format = sprintf( '%1$s %2$s', get_option( 'date_format', 'Y-m-d' ), get_option( 'time_format', 'g:i a' ) ); ?>
+			<strong><?php echo get_date_from_gmt( date( 'Y-m-d H:i:s', wp_next_scheduled( 'el_scheduled_delete_logs' ) ), $date_time_format ); ?></strong>
```
Comment (sudar) [inline include/Util/helper.php]: `wp_next_scheduled('el_scheduled_delete_logs')` can also be of type `false`; however, parameter $timestamp of `date()` accepts only integer. We need to add an additional type check.

**Comment by sudar** [comment]
@mariadanieldeepak 

Left some review feedback for you.

File: include/Util/helper.php (hunk 1 above)
Comment (mariadanieldeepak) [reply include/Util/helper.php]: @sudar The condition is already added on [L#143](https://github.com/sudar/email-log/pull/164/files/a7b5dd188c4ec640d1469c1ce938dd4a2a3809a8#diff-55d21dc299c734208c354fbbf4ed67a8R143)

**Pull request merged**